torch/nn/init.py

```diff
@@ -102,6 +103,8 @@ def calculate_gain(nonlinearity, param=None):
         else:
             raise ValueError("negative_slope {} not a valid number".format(param))
         return math.sqrt(2.0 / (1 + negative_slope ** 2))
+    elif nonlinearity == 'selu':
+        return 3.0 / 4
```

Comment:
Given that this value is non-standard, I'd add a saying that this has been empirically found and link to this PR for details.
Other than that, it looks good to me.
(Changing the other values probably would need to hook into the versioning scheme we don't have for init #41638, so it's good that you have not touched the old bits.)

Reply:
Also FB requires a CLA for contributions these days...

Reply:
@t-vi , @ngimel I have added the comment referencing this PR  in the return line, and I have signed the CLA.